src/service/Users/users.js

```diff
@@ -0,0 +1,30 @@
+const dbConnection = require('../../utils/dbConnection');
+const User = require('../../model/User');
+
+class Users {
```

Comment:
I think we should not write similar kind of crud for all the models.
if models are different then we should pass those models and all the other operations are similar this will bring the crud operations code at single place and if we need to update logic in any of the case then that is possible